intellst-api/src/Services/IdentifiedCaseHandler.php

```diff
@@ -81,4 +102,92 @@ public function updateIdentifiedCaseAllowEntrance(AllowEntranceDTO $dto, Identif
 
         return $errors;
     }
+
+    public function getIdentifiedCase(): array
+    {
+        $user = $this->userHandler->getList();
+        $temperature = $this->returnUserEnterprise($user)->temperature;
+        $data = $this->returnUserEnterprise($user)->restrictionPeriod;
+        $users = $this->identifiedCaseRepository->getNewIdentifiedCase($data, $temperature);
+        $arr = [];
+        foreach ($users as $identifiedCase) {
+            $identifiedCaseDTO = $this->identifiedCaseTransformer->transformEntityToDTO($identifiedCase);
+            $arr[] = $identifiedCaseDTO;
+        }
+
+        return $arr;
+    }
+
+    public function getReturnAttempts(IdentifiedCaseDTO $dto): bool
```

Comment:
why `getReturnAttempts` if it returns a boolean? `it should be `isReturnAttempt`